.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: node_js
+nodejs:
+  - node
+services:
+  - postgresql
+
+install:
+  - npm install
+
+script:
+  # - npm run test
+
+notifications:
+  slack: andela:E6PvFgt7ygC8QJHxXQJ7mEML
```

Comment:
Please create an environmental variable for this. This variable should not be exposed

Reply:
maybe a SLACK_ENV

Reply:
@benfluleck I have encrypted this string